talos.nf

```diff
@@ -0,0 +1,235 @@
+
```

Comment:
The general format for writing a DSL2 workflow in Nextflow now is to have each process code in a separate file under a `modules` directory, and then import the needed modules at runtime.
e.g. https://github.com/nf-core/smrnaseq/blob/72c4c4cf83e1f534319493f9de8344854f59736f/subworkflows/nf-core/bam_sort_stats_samtools/main.nf#L5-L7

Reply:
Yee nice, I'd forgotten about this. I have a local modules dir but wanted to get the syntax right before breaking it up